enterprise/server/backends/pebble_cache/pebble_cache_test.go

```diff
@@ -2165,6 +2167,148 @@ func TestDeleteEmptyDirs(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestUnspecifiedActiveKeyVersion_NewDatabase(t *testing.T) {
+	te := testenv.GetTestEnv(t)
+	te.SetAuthenticator(testauth.NewTestAuthenticator(emptyUserMap))
+	ctx := getAnonContext(t, te)
+	rootDir := testfs.MakeTempDir(t)
+	maxSizeBytes := int64(1_000_000) // 1MB
+	options := &pebble_cache.Options{RootDirectory: rootDir, MaxSizeBytes: maxSizeBytes}
+
+	// Confirm that a newly-created pebble database with an unspecified key
+	// version is written at the highest available key version.
+	activeKeyVersion := int64(filestore.UnspecifiedKeyVersion)
+	options.ActiveKeyVersion = &activeKeyVersion
+	pc := openPebbleCache(ctx, t, te, options, []string{"remote-instance-name-1"})
+	versionMetadata, err := pc.DatabaseVersionMetadata()
+	require.NoError(t, err)
+	assert.Equal(t, int64(filestore.MaxKeyVersion)-1, versionMetadata.MinVersion)
```

Comment:
Yeah, I noticed that, but thought it'd be better to start a trend of moving to `assert` for assertions rather than being consistent with the rest of the file. If you feel strongly I can change it, though.